app-admin/lsat/files/lsat-0.9.8.2-segfault-fix.patch

```diff
@@ -0,0 +1,36 @@
+--- lsat-0.9.7.1/lsatmain.c.orig	2008-12-29 11:09:04.000000000 -0500
```

Comment:
Where is this patch from? Was it submitted upstream?

Reply:
This is the same patch as the previous release, (See: https://bugs.gentoo.org/184488). It just has been tweaked to apply to the latest release, (Removed the `close((int)infile);` from [files/lsat-0.9.7.1-segfault-fix.patch](https://github.com/gentoo/gentoo/blob/master/app-admin/lsat/files/lsat-0.9.7.1-segfault-fix.patch)).

I raised an issue upstream, (See: https://github.com/triode3/lsat/issues/1).